build-tests-subspace/rush-sdk-test/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rush-sdk-test",
+  "version": "0.0.0",
+  "private": true,
+  "description": "A minimal example project that imports APIs from @rushstack/rush-sdk",
+  "license": "MIT",
+  "scripts": {
+    "build": "heft build --clean",
+    "start": "node lib/start.js",
+    "_phase:build": "heft run --only build -- --clean"
+  },
+  "dependencies": {
+    "@rushstack/rush-sdk": "workspace:*"
+  },
+  "devDependencies": {
+    "@rushstack/heft": "workspace:*",
+    "@rushstack/heft-lint-plugin": "workspace:*",
```

Comment:
These projects need `.eslintrc.js` files.

It'd also be ideal to point those at `local-eslint-config`.

Reply:
Ok, I switched to rig.json to manage these configs